engine/core/src/main/java/org/datacleaner/metamodel/datahub/utils/DataHubColumnBuilder.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.metamodel.datahub.utils;
+
+import org.apache.metamodel.schema.Column;
+import org.apache.metamodel.schema.ColumnType;
+import org.apache.metamodel.schema.MutableColumn;
+import org.apache.metamodel.schema.MutableTable;
+import org.apache.metamodel.schema.Table;
+
+public class DataHubColumnBuilder {
+    String _name;
+    ColumnType _type;
+    Integer _number;
+    Integer _size;
+    String _nativeType;
+    boolean _nullable;
+    String _remarks;
+    boolean _indexed;
+    String _quote;
+    boolean _primaryKey;
+    Table _table;
+
+    public DataHubColumnBuilder withName(String name) {
+        _name = name;
+        return this;
+    }
+
+    public Column build() {
+        final MutableColumn column = new MutableColumn(_name, _type, _table, _number, _size, _nativeType, _nullable,
```

Comment:
I think my previous remark was a bit misunderstood. What I meant wasn't that ImmutableColumn is bad, but that this builder class could easily be replaced by just using a MutableColumn instead. Not that there's anything wrong with the builder - it just seemed to me to be quite generic and not DataHub related so I thought "it shouldn't be necesary for the DH guys to invent a builder to do this common thing".